### PR TITLE
Allow referencing message as argument to `!remind edit content`

### PR DESCRIPTION
**Pull request by TizzySaurus**
Part two of #2231.

Users can now reference a message to use its content as the reminder's new content when editing a reminder, as is already the case when creating a reminder.

*Text argument*
image

*Referenced message has a content*
image

*Referenced message has no content*
image

*No text and no referenced message*
image

### CODE REVIEW

File: bot/exts/utils/reminders.py [hunk 1]
```diff
@@ -209,6 +209,29 @@ async def send_reminder(self, reminder: dict, expected_time: t.Optional[time.Tim
...
+                content = resolved_message.content
+
+        # If we weren't able to get the content of a replied message
+        if content is None:
+            await send_denial(ctx, "Your reminder must have a content and/or reply to a message.")
+            return
+
+        # If the replied message has no content (e.g. only attachments/embeds)
+        if content == "":
+            content = "See referenced message."
```
Comment (mbaruh) [inline bot/exts/utils/reminders.py:231]: Can you put this in italics? to better separate it from a message's content.

**Review by mbaruh — CHANGES_REQUESTED** [review]

File: bot/exts/utils/reminders.py (hunk 1 above)
Comment (TizzySaurus) [reply bot/exts/utils/reminders.py]: Yep, done :+1:

**Review by wookie184 — CHANGES_REQUESTED** [review]
Works nicely. Could you mention the behaviour in the docstrings?

**Comment by TizzySaurus** [comment]
> Works nicely. Could you mention the behaviour in the docstrings?

Done!

**Review by vivekashok1221 — APPROVED** [review]
Reviewed and tested.
LGTM :star2:

**Review by wookie184 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**